src/components/range-slider/range-slider.tsx

```diff
@@ -70,7 +72,7 @@ export const RangeSlider = ({
     };
 
     return (
-        <div>
+        <div className={classNames('rangeSlider', className)}>
```

Comment:
I meant to pass `rangeSlider` to the instance. So this entire folder could be copied into a different project with no changes.

Ideally the component would come with some default hardcoded colors:
```
// range-slider.module.scss
.root {
  --thumb-color: #000;
}
```

But this would make overriding default styling tricky, since `common.scss` appears in the document earlier than `range-slider.module.scss`. But we could use `:where()` to control specificity:

https://jsfiddle.net/o94a8pn0/

Reply:
Done.
Plus found this bug in Codux. It says value is overriden when it is not.

<img width="298" alt="Screenshot 2024-10-02 at 15 38 27" src="https://github.com/user-attachments/assets/6a45f671-61ba-4e00-b43f-e4682a050982">

Reply:
I guess because the specificity calculator doesn't understand `:where`.